clients/drcachesim/CMakeLists.txt

```diff
@@ -168,6 +168,11 @@ add_exported_library(drmemtrace_func_view STATIC tools/func_view.cpp)
 add_exported_library(drmemtrace_invariant_checker STATIC tools/invariant_checker.cpp)
```

Comment:
Please see https://dynamorio.org/page_code_reviews.html#sec_commit_messages on conventions on PR description title and body.

Reply:
I see. So should I open a new issue to acquire an issue number?

Reply:
Yes, as I don't think one exists yet.